HW_9/decorators.py

```diff
@@ -0,0 +1,42 @@
+# Декоратор, запускающий функцию нахождения корней квадратного уравнения
+# с каждой тройкой чисел из csv файла.
+# Декоратор, сохраняющий переданные параметры и результаты работы функции в json файл.
+
+
+import csv
+import json
+from functools import wraps
+
+
+def use_csv_data(csv_file):
+    def deco(func):
+        res = []
+
+        @wraps(func)
+        def wrapper(*args, **kwargs):
+            with open(csv_file, 'r', newline='') as f:
+                reader = csv.reader(f)
+                next(reader)
+                for row in reader:
+                    args = map(int, row)
+                    res.append(func(*args, **kwargs))
```

Comment:
```suggestion
                    res.append(func(*map(int, row)))
```
Однако по условию, этот декоратор должен только вызывать функцию. Ничего остального он делать не должен